src/SBarcodeGenerator.cpp

```diff
@@ -0,0 +1,64 @@
+#include <QStandardPaths>
+#include "SBarcodeGenerator.h"
+#include "MultiFormatWriter.h"
+#include "TextUtfEncoding.h"
+
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include "stb_image_write.h"
+
+SBarcodeGenerator::SBarcodeGenerator()
+{
+
+}
+
+bool SBarcodeGenerator::process(const QString inputString)
+{
+    if (inputString.isEmpty()){
+        return 0;
+    }
+    else {
```

Comment:
This method returns boolean. Not integer
Also there is no need to write else. Just write:

```
if (inputString.isEmpty()){
  return false;
}

//your code
return true;
```